workflow-steps/install-node-modules/main.js

```diff
@@ -69,3 +102,5 @@ function patchJest() {
     console.log('no need to patch jest');
   }
 }
+
+main();
```

Comment:
I wonder if we could add a `fetch-retries` just for NPM and yarn, as they might retry at the dep level rather than the whole thing? Optional though, it probably requires some extra testing etc.

Reply:
we can look at configuring this for npm and pnpm via npmrc file. I think that would have to change in the base image where we put that config setting globally instead of local in the workspace to prevent overwriting anyones custom npmrc file?